frontend/workflows/projectSelector/src/dash.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+
+import { DashDispatchContext, DashStateContext } from "./dash-hooks";
+import ProjectSelector from "./project-selector";
+import type { DashAction, DashState } from "./types";
+
+const initialState = {
+  selected: [],
+  projectData: {},
+};
+
+const dashReducer = (state: DashState, action: DashAction): DashState => {
+  switch (action.type) {
+    case "UPDATE_SELECTED": {
+      return action.payload;
+    }
+    default:
+      throw new Error("not implemented (should be unreachable)");
+  }
+};
+
+const Dash = ({ children }) => {
+  const [state, dispatch] = React.useReducer(dashReducer, initialState);
+
+  return (
+    <DashDispatchContext.Provider value={dispatch}>
+      <DashStateContext.Provider value={state}>
+        <ProjectSelector />
+        <div>{children}</div>
```

Comment:
card has grid styling so this div affects that
```suggestion
        {children}
```

Reply:
hmm i did this so the cards wouldn't be siblings with the selector

Reply:
ah ok. we can make style updates to the card

Reply:
i probably need to flex grow this div, let me try that